### PR TITLE
release 1.3.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
I will wait with this until https://github.com/voxpupuli/modulesync/pull/182 got merged

**Comment by bastelfreak** [comment]
lets wait for https://github.com/voxpupuli/modulesync/pull/181 / https://github.com/voxpupuli/modulesync/pull/183

**Comment by raphink** [comment]
And https://github.com/voxpupuli/modulesync/pull/184 if possible (and then I think I'm done for this time 😁)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # Changelog
 
+## 2020-07-03 - 1.3.0
+
+* Expose --managed_modules_conf [#184](https://github.com/voxpupuli/modulesync/pull/184)
+* https://github.com/voxpupuli/modulesync/pull/183 [#183](https://github.com/voxpupuli/modulesync/pull/183)
```
Comment (raphink) [inline CHANGELOG.md:6]:
```suggestion
* Allow absolute path for config files [#183](https://github.com/voxpupuli/modulesync/pull/183)
```

**Pull request merged**